boards/nxp/mcxw72_evk/doc/index.rst

```diff
@@ -0,0 +1,146 @@
+allowing firmware updates to support future wireless standards.
+
+Hardware
+********
+
+- MCXW72 Arm Cortex-M33 microcontroller running up to 96 MHz
+- 2MB on-chip Flash memory unit
+- 256 KB TCM RAM
+- On-board MCU-Link debugger with CMSIS-DAP
+
+For more information about the MCXW72 SoC and MCXW72-EVK board, see:
+
+- `MCXW72 SoC Website`_
+- `MCXW72-EVK Website`_
+
+Supported Features
+==================
+
+The ``mcxw72_evk`` board target in Zephyr currently supports the following features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinctrl                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| LPUART    | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| TPM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| LPTMR     | on-chip    | counter                             |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| FLEXCAN   | on-chip    | can                                 |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the MCU-Link CMSIS-DAP Onboard Debug Probe.
+
+Using J-Link
+------------
+
+There are two options. The onboard debug circuit can be updated with Segger
+J-Link firmware by following the instructions in
+:ref:`mcu-link-jlink-onboard-debug-probe`.
+To be able to program the firmware, you need to put the board in ``DFU mode``
+by shortening the jumper JP20.
+The second option is to attach a :ref:`jlink-external-debug-probe` to the
+10-pin SWD connector (J16) of the board.
+For both options use the ``-r jlink`` option with west to use the jlink runner.
```

Comment:
maybe `-r jlink` is not needed since that is the only supported runner, or we can keep this if we support linkserver later I guess

Reply:
> maybe `-r jlink` is not needed since that is the only supported runner, or we can keep this if we support linkserver later I guess

Probably, Linkserver should be a default runner as for mcxw71

Reply:
LinkServer is now supported and the default runner.